test/web-platform-tests/to-run.yaml

```diff
@@ -171,7 +172,7 @@ throw-on-dynamic-markup-insertion-counter-reactions.html: [timeout, document.ope
 
 DIR: dom/abort
 
-event.any.html: [fail, AbortSignal.abort() not implemented]
+AbortSignal.any.html: [fail, AbortSignal.abort() not implemented]
```

Comment:
We can probably implement this pretty easily. The spec is here: https://dom.spec.whatwg.org/#dom-abortsignal-abort

Just add a `abort()` static method here: https://github.com/jsdom/jsdom/blob/0024630fc62d1abb567cd531f9396b9a811424aa/lib/jsdom/living/aborting/AbortSignal-impl.js#L7

and update the [IDL file](https://github.com/jsdom/jsdom/blob/0024630fc62d1abb567cd531f9396b9a811424aa/lib/jsdom/living/aborting/AbortSignal.webidl#L2) to conform to the [spec](https://dom.spec.whatwg.org/#abortsignal).

You might need to use [[WebIDL2JSCallWithGlobal]](https://github.com/jsdom/webidl2js#webidl2jscallwithglobal) in the IDL file to make sure we can create the new AbortSignal in the right realm.

----

Here's the equivalent Chromium implementation: https://chromium-review.googlesource.com/c/chromium/src/+/2936679

Reply:
So, this requires creating an impl+wrapper from inside the implementation class, which is ordinarily done with the generated file's `createImpl()`. I tried implementing this with a circular `require()`, which seems? to work, but might not be a great idea. Is there a known, better way to do this?

Reply:
It looks a little strange, but indeed that's how it's supposed to work. Looks good!